pkgs/collection/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: collection
-version: 1.19.1
+version: 1.19.1-wip
```

Comment:
Should be 1.20.0, unless 1.19.0 hasn't been released yet.
(Adds new feature &rArr; minor version increment.)